openedx/core/djangoapps/content/course_overviews/migrations/0008_drop_course_tabs_fk_constraint.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        db.delete_foreign_key('course_overviews_courseoverviewtab', 'course_overview_id')
```

Comment:
@nasthagiri: this is what I ended up doing -- deleting the fkey constraint, but leaving behind the column. The remaining code seems to work fine. I tested having a bunch of v2 data after this migration, rolling back to release, having it regenerate all the course overviews on my dashboard, then going back to v2 code and having it re-generate all v2 data. Used the admin dashboard in between to verify that v1 writes and then v2 writes had happened.